ingress/single-cluster/ingress-cloudarmor/cleanup.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit;
+set -o nounset;
+set -o pipefail;
+set -o xtrace;
+
+source "$(find -name helper.sh)" "$@"
```

Comment:
Why are we passing all of the args to this script directly to the helper.sh
and test.conf?

Reply:
helper.sh takes two kinds of arguments, one kind is specific to the test we are running, the other is specific to the variables in test.conf. 
And since variables test.conf could be generated dynamically(with different global variables), it would allow users to run test with different configurations.